db/schema.rb

```diff
@@ -157,19 +139,16 @@
     t.datetime "deleted_at"
     t.string "detailable_type"
     t.bigint "detailable_id"
-    t.uuid "appointment_reminder_id"
-    t.index ["appointment_id"], name: "index_communications_on_appointment_id"
-    t.index ["appointment_reminder_id"], name: "index_communications_on_appointment_reminder_id"
+    t.uuid "notification_id"
+    t.uuid "appointment_id"
+    t.uuid "patient_id"
```

Comment:
I no longer think we need `patient_id` on communication, because notifications has all that data.

Reply:
I think you're right.